varnish/tests/test_e2e.py

```diff
@@ -0,0 +1,14 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from . import common
+
+
+@pytest.mark.e2e
+def test_check(dd_agent_check, instance):
+    aggregator = dd_agent_check(instance, rate=True)
+    for mname in common.COMMON_METRICS:
+        aggregator.assert_metric(mname, tags=['cluster:webs', 'varnish_name:default'])
```

Comment:
assert all metrics covered?

Reply:
No there are way too many metrics. I don't know why they are not covered by the integration tests, maybe we can look later?